curvefs/src/mds/fs_manager.cpp

```diff
@@ -793,5 +871,49 @@ void FsManager::CommitTx(const CommitTxRequest* request,
     dlock_->UnLock(fsName, uuid);
 }
 
+void FsManager::UpdateClientAliveTime(const Mountpoint& mountpoint,
+    const std::string& fsName, bool remove) {
+    VLOG(1) << "UpdateClientAliveTime fsName = " << fsName
+            << ", mp = " << mountpoint.DebugString() << ", remove = " << remove;
+    std::string mountpath;
+    MountPoint2Str(mountpoint, &mountpath);
+    WriteLockGuard wlock(recorderMutex_);
+    if (remove) {
+        auto iter = mpTimeRecorder_.find(mountpath);
+        if (iter == mpTimeRecorder_.end()) {
+            LOG(WARNING) << "remove client record ttl but not exist, "
+                         << "fsName = " << fsName
+                         << ", mountpoint = " << mountpath;
+            return;
+        }
+        mpTimeRecorder_.erase(iter);
+        return;
+    }
+
+    mpTimeRecorder_[mountpath] =
```

Comment:
`mpTimeRecorder_` is recorded by using `UpdateClientAliveTime`. Consider the following situation:
1. The client mout succeeded. After a period of time, mds hangs up, and the client also exits abnormally.
2. mds is up
Exception client will not be cleaned up. `mpTimeRecorder_`  should load from mds mountpoint at the begining.

Reply:
fixed